src/utils/api.js

```diff
@@ -51,3 +51,7 @@ export function postComment(article_id, username, body) {
       throw new Error("Failed to post comment");
     });
 }
+
+export function deleteCommentByCommentId(comment_id) {
```

Comment:
You can slightly rename this to deleteCommentById - it's safe to assume that if you're mentioning just the id then it will be the comment id